src/transactions/CreateClaimableBalanceOpFrame.cpp

```diff
@@ -14,6 +14,19 @@
 namespace stellar
 {
 
+void
+setClaimableBalanceClawbackEnabled(ClaimableBalanceEntry& cb)
```

Comment:
this should be static/anonymous namespace as it's only used here

Reply:
Done. I added another commit that did the same for the other free functions here.